grantnav/api/aggregates.py

```diff
@@ -11,7 +11,13 @@ def get(self, *args, **kwargs):
         # Append .aggregates_api as this is the flag we currently switch on in the
         # main search functions to know what format we will return results as.
         self.request.path = self.request.path + ".aggregates_api"
-        context = search(self.request)
+        ret = search(self.request)
```

Comment:
A view calling another view :smile: Is this a consequence of the backends merge? that `search` looks real hairy, doubt we'll be refactoring that anytime soon.

Reply:
Yeah, it's overdue some rationalisation, it's not quite as scary as it looks and this can be thought of a wrapper around the search request, with a minimal amount of duct tape to avoid making the search function larger and to start the process of splitting it up.